README.md

```diff
@@ -22,7 +22,7 @@ jobs:
     name: Check for story ID
     runs-on: ubuntu-latest
     steps:
-      - uses: movableink/pr-clubhouse-lint-action@release
+      - uses: movableink/pr-shortcut-lint-action@release
```

Comment:
We arent actually updating this so this should remain clubhouse. As it impacts all repos.
```suggestion
      - uses: movableink/pr-clubhouse-lint-action@release
```